src/pages/Admin/Venue/DetailsForm.tsx

```diff
@@ -805,8 +841,67 @@ const DetailsFormLeft: React.FC<DetailsFormLeftProps> = ({
     </div>
   );
 
+  const { sovereignVenueId } = useSovereignVenueId({ venueId });
+
+  const { currentVenue: venue } = useConnectCurrentVenueNG(sovereignVenueId);
+
+  const [userStatuses, setUserStatuses] = useState<UserStatus[]>([]);
+
+  const {
+    isShown: hasUserStatuses,
+    hide: hideUserStatuses,
+    show: showUserStatuses,
+  } = useShowHide(venue?.showUserStatus);
+
+  useEffect(() => {
+    if (!venue) return;
+    const venueUserStatuses = venue?.userStatuses ?? [];
+
+    setUserStatuses(venueUserStatuses);
+
+    if (venue.showUserStatus) {
+      showUserStatuses();
+    }
+  }, [showUserStatuses, venue]);
+
+  const removeUserStatus = (index: number) => {
+    const statuses = [...userStatuses];
+    statuses.splice(index, 1);
+    setUserStatuses(statuses);
+  };
+
+  const toggleUserStatus = useCallback(() => {
+    hasUserStatuses ? hideUserStatuses() : showUserStatuses();
```

Comment:
can we use `toggle` from `useShowHide` here instead?

Reply:
Fixed in: https://github.com/sparkletown/sparkle/pull/1532/commits/2de383019a9ecbef3edfbcc744a06df2a0ffac64